src/System.Management.Automation/engine/ArgumentTypeConverterAttribute.cs

```diff
@@ -152,11 +152,6 @@ internal object Transform(EngineIntrinsics engineIntrinsics, object inputData, b
                         if (_convertTypes[i] == typeof(ActionPreference))
```

Comment:
@KirkMunro, your last commit had 4 failures in `PowerShell-CI-windows`
Type inference Tests.Infers type from block statement
```powershell
Expected 'System.Int32', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Api\TypeInference.Tests.ps1: line 600
600:         $res.Name | Should -Be 'System.Int32'
```
PowerShellGet - Script tests (Admin).Should install a script correctly to the required location with AllUsers scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PowerShellGet\PowerShellGet.Tests.ps1: line 234
234:         $installedScriptInfo | Should -Not -BeNullOrEmpty
```
PowerShellGet - Script tests.Should install a script correctly to the required location with default CurrentUser scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PowerShellGet\PowerShellGet.Tests.ps1: line 207
207:         $installedScriptInfo | Should -Not -BeNullOrEmpty
```
Runspace Debugging API tests.PSStandaloneMonitorRunspaceInfo tests.Error occurred in Context block
```powershell
Cannot convert argument "runspaceType", with value: "", for ".ctor" to type "System.Management.Automation.Internal.PSMonitorRunspaceType": "Cannot convert null to type "System.Management.Automation.Internal.PSMonitorRunspaceType" due to enumeration values that are not valid. Specify one of the following enumeration values and try again. The possible enumeration values are "Standalone,InvokeCommand"."
At D:\a\1\s\test\powershell\SDK\PSDebugging.Tests.ps1:195 char:13
```